gcp_variant_transforms/pipeline_common.py

```diff
@@ -131,6 +132,18 @@ def get_pipeline_mode(all_patterns, optimize_for_large_inputs=False):
     return PipelineModes.MEDIUM
   return PipelineModes.SMALL
 
+def get_estimates(pipeline, pipeline_mode, all_patterns):
+  # type: (beam.Pipeline, int, List[str]) -> pvalue.PCollection
+  """Creates a PCollection by reading the VCF files and deriving estimates."""
```

Comment:
Since we just headers and one data line from the files, does it make sense to have a different pipeline when PipelineModes.LARGE? Or in other words, if PipelineModes.LARGE, and we use pipeline | vcf_estimate_io.GetEstimates(all_patterns[0]), is there any downside?

Reply:
well it will fail if we have input file with multiple patterns, so at least for that case we do need GetAllEstimates.

However, even if that wasn't the case, I'm not sure if there is much benefit of removing it - I mean, even if the number of header lines in the input files is small, wouldn't this still improve the speed if the number of files is very large (albeit a little)? What are we losing here, readability? We already have single vs all in other reads, so I don't think it's that confusing...